modules/virt-automatic-certificates-renewal.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/node_maintenance/virt-automatic-certificates.adoc
+
+[id="virt-automatic-certificates-renewal_{context}"]
+= Automatic renewal of TLS certificates
+
+TLS certificates for {VirtProductName} are created during installation for these components:
+
+* KubeVirt
+* containerized data importer controller(CDI)
+* MAC pool
```

Comment:
@ksimon1 anything to add there for SSP?
@jelkosz anything to add there for the web ui?

Reply:
For SSP, only the template-validator uses a certificate. It is generated and rotated by OpenShift, and
mounted as a secret to the pod. I'm not sure how relevant it is for the documentation.

Reply:
@rmohr @stu-gott  - Please advise if all angles have been explored, and if this PR can move to QE review stage?  After QE review the content goes to peer review stage in order to meet the deadline for wrapping up PRs for 2.4 release. Thanks.